halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/cloudrun/CloudrunCommandProperties.java

```diff
@@ -0,0 +1,46 @@
+package com.netflix.spinnaker.halyard.cli.command.v1.config.providers.cloudrun;
+
+public class CloudrunCommandProperties {
+  static final String LOCAL_REPOSITORY_DIRECTORY_DESCRIPTION =
+      "A local directory to be used to stage source files"
+          + " for Cloud Run deployments within Spinnaker's Clouddriver microservice.";
+  static final String GIT_HTTPS_USERNAME_DESCRIPTION =
```

Comment:
Is it useful to cloudrun? What is the usecase with GIT?

Reply:
It is not useful for cloudrun. we need to remove.

Reply:
@kirangodishala Can you review once now?

Reply:
looks fine.